src/common/utils/elevation.h

```diff
@@ -83,9 +83,25 @@ namespace
     inline bool GetDesktopAutomationObject(REFIID riid, void** ppv)
     {
         CComPtr<IShellView> spsv;
-        if (!FindDesktopFolderView(IID_PPV_ARGS(&spsv)))
+        
+        // Desktop may not be available on startup
+        auto attempt = 0;
+        while (true)
         {
-            return false;
+            if (attempt > 9)
```

Comment:
Why not while attempts less than 9 vs true?

Reply:
The `if` was intended for `return false`.
I will improve it a bit.

Reply:
just a suggestion, i tend to not like while loops personally cause a small tweak can fubar them without realizing it.